### DISCUSSION TITLE
Any plans to add Metal API support on Macs?

### DISCUSSION THREAD

**Discussion by VadimMuhtarov** [dbeaver/dbeaver#23146] [General]
Hi. Do you plan to add the Metal API support for UI via "java -Dsun.java2d.metal=true" flag? Because it's available since JDK 17.

**Comment by LonwoLonwo** [comment]
Hello @VadimMuhtarov 

Please explain more - why do we need to add it?

**Reply by VadimMuhtarov** [reply]
You can check info here https://openjdk.org/jeps/382 
Idea and Android Studio feels much smoother with it for example

**Reply by LonwoLonwo** [reply]
Looks like it is more for AWT. However, we are not using AWT but SWT for our project.
